extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -482,6 +501,20 @@ export class AmpSlideScroll extends BaseSlides {
   }
 
   /**
+   * Shows the slide at the given index and triggers a `slidechange` action.
+   * @param {number} newIndex
+   * @private
+   */
+  showSlideAndTriggerAction_(newIndex) {
```

Comment:
see if we can trigger this action on `goCallback` in base-slides.js and then we will be able to avoid this wrapper completely.